src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -348,10 +376,25 @@ const styles = ({breakpoints, palette, spacing}) => ({
       color: 'black',
     },
   },
+  alignButtonsBetween: {
+    justifyContent: 'space-between',
+  },
+  alignButtonsEnd: {
+    justifyContent: 'flex-end',
+  },
```

Comment:
It's kind of awkward you have to switch between these two. I think you can always use `space-between` if you put an empty element in above instead of `null` for the delete button

Reply:
Use space-between with <span /> now. Thanks. Done.